src/app/options-modal/options-modal.component.html

```diff
@@ -22,6 +22,14 @@
       </select>
     </span>
   </div>
+  <div *ngIf="homeService.autoBuyLandUnlocked || homeService.autoBuyHomeUnlocked || homeService.autoBuyFurnitureUnlocked" class="optionField">
```

Comment:
Appears below auto-buy options if any auto-buy manual has been purchased. For furniture, this looks a little weird since it doesn't actually add any options.